app/assets/stylesheets/layouts/river.scss

```diff
@@ -1,35 +1,118 @@
-#logo { background-color: #434D73; }
-#nav-top, #user-info { background-color: #212135; }
-#header, #nav-bottom { background-color: #ADB5B9; }
-.post-expander { background-color: #A4B0BD; }
-.pagination .current { background: #718EB9; }
-.post-icon, #current-icon-dropdown, #reply-icon-selector, .replace-icon, .icons-box, .icon-credit { background-color: #A9B4BB; }
-a { color: #3F5970; }
-a:visited { color: #284150; }
-.flash.breadcrumbs, .form-table-ender { background-color: #A3A8A9; }
-.subber { background-color: #B9B6AD; }
+$bg_color_header_logo: #212135; // top header strip for logo
+$bg_color_header_navbottom: #ADB5B9;
+$bg_color_header_middle: #212135; // middle header strip with user items
+$bg_color_header_bottom: #ADB5B9; // bottom header strip with navigation
+
+#header { background-color: $bg_color_header_bottom; }
+#logo { background-color: $bg_color_header_logo; }
+#nav-bottom { background-color: $bg_color_header_navbottom; }
+#nav-top, #user-info { background-color: $bg_color_header_middle; }
+
+/* - links */
+$font_color_link: #3F5970; // link
+$font_color_link_visited: #284150;
+//$font_color_link_hover: #FCFEFD; // link (hovered)
+a { color: $font_color_link; }
+a:visited { color: $font_color_link_visited; }
+
+$bg_color_flash_breadcrumbs: #A3A8A9; // breadcrumbs ("Continuities » Continuity » Section » Thread")
+.flash.breadcrumbs { background-color: $bg_color_flash_breadcrumbs; }
+
+$bg_color_expander: #A4B0BD; // expander on posts
+.post-expander { background-color: $bg_color_expander; }
+
+$bg_color_page_current: #718EB9;
+.pagination .current { background-color: $bg_color_page_current; }
+
+$bg_color_post_icon: #A9B4BB;
+$bg_color_icon_selector: #A9B4BB;
+//$font_color_icon_selector: #111111;
+
+.post-icon { background-color: $bg_color_post_icon; }
+#current-icon-dropdown { background-color: $bg_color_post_icon; }
+#reply-icon-selector { background-color: $bg_color_icon_selector; }
+
+/* - icons */
+//$border_color_icon_selected: #B9BBEB; // selected icon border
+$bg_color_icon: #A9B4BB;
+$bg_color_icon_credit: #A9B4BB;
+
+.replace-icon, .icons-box  { background-color: $bg_color_icon; }
+.icon-credit { background-color: $bg_color_icon_credit; }
+
+$bg_color_subber1: #A3A8A9; // tertiary content header
+$bg_color_subber2: #B9B6AD; // tertiary content header // TODO: EXAMINE THIS
+
+.form-table-ender { background-color: $bg_color_subber1; }
+.subber { background-color: $bg_color_subber2; }
+
+$bg_color_linkbox_new: #3F5370;
+.link-box.action-new { background-color: $bg_color_linkbox_new; }
+
+/* - content */
+$bg_color_even: #F3F3F3;
+$bg_color_odd: #E8E8E8;
+$bg_color_content_border: #aaa;
+
+#content .odd, #content .even, .post-post, .post-reply, #post-editor {
+  background-color: $bg_color_even !important;
+  border-top: solid 1px $bg_color_content_border;
+  blockquote { background-color: calculate-transparent-color($bg_color_even, $bg_color_odd, 0.2) !important; }
+} // TODO: Why the heck is even and odd set the same
```

Comment:
because it has a border to distinguish the gap, and timepoof didn't want an even/odd distinction on the page.

Reply:
yeah, I think you said in chat, lemme clean that up